### ISSUE TITLE
Newt SHA256 mismatch

### ISSUE THREAD

**Issue by yawn** [Homebrew/homebrew-core#5787]
- [X] Ran `brew update` and retried your prior step?

In the following `brew` version:

```
HOMEBREW_VERSION: 1.0.7
ORIGIN: https://github.com/Homebrew/brew.git
HEAD: 0546d90e3dbdae30407e5256d680be5ed5f5f000
Last commit: 2 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: e130cbafee3cee086d517558d82068ae9fa8cde1
Core tap last commit: 2 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local/Homebrew
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: octa-core 64-bit ivybridge
Homebrew Ruby: 2.0.0-p648
GCC-4.2: build 5666
Clang: 8.0 build 800
Git: 2.10.0 => /usr/local/bin/git
Perl: /usr/bin/perl
Python: /usr/local/bin/python => /usr/local/Cellar/python/2.7.12_1/Frameworks/Python.framework/Versions/2.7/bin/python2.7
Ruby: /Users/.../.rvm/rubies/ruby-2.1.4/bin/ruby
Java: 1.8.0_60
macOS: 10.12-x86_64
Xcode: 8.0
CLT: N/A
X11: 2.7.7 => /opt/X11
```

The install command for the formula "[newt](https://github.com/Homebrew/homebrew-core/blob/e130cbafee3cee086d517558d82068ae9fa8cde1/Formula/newt.rb)" results in the following SHA256 mismatch:

```
brew install newt
==> Using the sandbox
==> Downloading https://fedorahosted.org/releases/n/e/newt/newt-0.52.18.tar.gz
==> Downloading from https://pagure.io/releases/newt/
######################################################################## 100,0%
Error: SHA256 mismatch
Expected: 771b0e634ede56ae6a6acd910728bb5832ac13ddb0d1d27919d2498dab70c91e
Actual: 5eee8a2be8006504ea554f91a6b4cade6f3f87ab6035d344fcf9b3140f3ef78a
Archive: /Users/.../Library/Caches/Homebrew/newt-0.52.18.tar.gz
To retry an incomplete download, remove the file above.
```

**Comment by zmwangx** [comment]
Upstream moved. You can use this patch before we fix it here:

```diff
diff --git a/Formula/newt.rb b/Formula/newt.rb
index 76d7300..53a3f48 100644
--- a/Formula/newt.rb
+++ b/Formula/newt.rb
@@ -1,7 +1,7 @@
 class Newt < Formula
   desc "Library for color text mode, widget based user interfaces"
-  homepage "https://fedorahosted.org/newt/"
-  url "https://fedorahosted.org/releases/n/e/newt/newt-0.52.18.tar.gz"
+  homepage "https://pagure.io/newt"
+  url "https://pagure.io/releases/newt/newt-0.52.18.tar.gz"
   sha256 "771b0e634ede56ae6a6acd910728bb5832ac13ddb0d1d27919d2498dab70c91e"
   revision 1
 
```

**Comment by zmwangx** [comment]
Fix in #5795.
